go/obscuronode/enclave/enclaverunner/cli_test.go

```diff
@@ -10,7 +10,7 @@ import (
 const testToml = "/test.toml"
 
 func TestConfigIsParsedFromTomlFileIfConfigFlagIsPresent(t *testing.T) {
-	expectedChainID := int64(1377) //nolint:ifshort
+	expectedChainID := int64(1377) //nolint
```

Comment:
nit: can this have the ignore lint reason ?

Reply:
Weirdly it can't :D If I put `//nolint:ifshort`, it complains that it's not used. If I remove the `nolint` entirely, it complains with an `ifshort`. 🤷